content/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot.md

```diff
@@ -445,8 +445,8 @@ This guide demonstrates how to get coding suggestions from {% data variables.pro
 {% data variables.product.prodname_copilot %} offers coding suggestions as you type. For example, type this function
 signature in a Swift file:
 
-```shell copy
-func CalculateDaysBetweenDates(
+```text copy
```

Comment:
I think that `shell` may have been a typo. It probably makes sense to make the following update, to align with examples for other IDEs.

```suggestion
```swift copy
```